lapis-e2e/test/details.spec.ts

```diff
@@ -91,6 +95,35 @@ describe('The /details endpoint', () => {
     expect(resultWithLimitAndOffset.data[0]).to.deep.equal(resultWithLimit.data[1]);
   });
 
+  it('should handle advancedQuery', async () => {
+    const urlParams = new URLSearchParams({
+      fields: 'primaryKey',
+      aminoAcidInsertions: 'ins_S:143:T,ins_ORF1a:3602:F?P',
+      division: 'Vaud',
+      orderBy: 'primaryKey',
+      dataFormat: 'csv',
+    });
+
+    const result = await fetch(basePath + '/sample/details?' + urlParams.toString());
```

Comment:
Why didn't you use `lapisClient.postDetails`?

usually, that's the preferred method, because it implicitly also tests our OpenAPI spec. But it has limitation so that it can't always be used.

Reply:
Did you regenerate the client after adding the `advancedQuery` field to the API? The generated client only let's properties pass that it knows. If you still use the client that doesn't know about that field, then it might be ignored. (IIRC)

Reply:
The advancedQueries param can be used just not the division.regex param - so I don't think this can be linked?